variables.tf

```diff
@@ -56,12 +58,12 @@ variable "origin_ssl_protocols" {
 }
 
 variable "origin_keepalive_timeout" {
-  description = "(Optional) The Custom KeepAlive timeout, in seconds. By default, AWS enforces a limit of 60. But you can request an increase."
+  description = "(Optional) - The Custom KeepAlive timeout, in seconds. By default, AWS enforces a limit of 60. But you can request an increase."
```

Comment:
Let's stick with the terraform formatting convention: (no `-` after `(Optional)`)

![image](https://user-images.githubusercontent.com/52489/30583786-cbafb21a-9cdc-11e7-9fd7-d437e1ff1b53.png)